### ISSUE TITLE
Using custom status as Completed

### ISSUE THREAD

**Issue by haogeek** [clearflask/clearflask#145]
Originally clearflask has a builtin completed status in roadmap task category, but if I delete it and use any other custom status as completed, then the mechanism "completed tasks can be added to announcement on next page" will break because announcement management does not know which status is "completed", so it will mistakely show tasks that can be linked to but with wrong status. I think you may want to add a config to setup a "completed" status just as the "entry status".

**Comment by matusfaro** [comment]
That's right. Just tested this and there isn't a way to re-create it right now (I thought when you disable and enable the roadmap it will, but it doesn't). When I have time I will add a button to do this.

**Comment by matusfaro** [comment]
@claude Can you please fix this? whenever you re-enable roadmap/feedback etc... I want you to re-create statuses if missing
